grails-app/migrations/views/changelog.xml

```diff
@@ -108,6 +108,9 @@
     <sqlFile path="views/order-item-details.sql"/>
   </changeSet>
   <!-- views required for cycle count last counted -->
+  <changeSet id="1740696149645-0" author="kchelstowski" runOnChange="true" runAlways="true" failOnError="true">
+    <sqlFile path="views/product-inventory-candidate.sql"/>
+  </changeSet>
```

Comment:
this has to be placed at the top of the sub-views, as the `product-physical-count-history` relies on this.